project/Dependencies.scala

```diff
@@ -19,14 +19,15 @@ object Dependencies {
     crossScalaVersions := Seq("2.11.8"), // "2.12.0-M4"
     scalaVersion := crossScalaVersions.value.head,
     scalaStmVersion := sys.props.get("akka.build.scalaStmVersion").getOrElse("0.7"),
-    scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.11.6"),
+    scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.13.2"),
     scalaTestVersion := {
       scalaVersion.value match {
         case "2.12.0-M1" => "2.2.5-M1"
         case "2.12.0-M2" => "2.2.5-M2"
         case "2.12.0-M3" => "2.2.5-M3"
         case "2.12.0-M4" => "2.2.6"
-        case "2.12.0-M5" => "3.0.0-RC4"
+        case "2.12.0-M5" => "3.0.0"
+        case "2.11.8"  => "3.0.0"
```

Comment:
I don't get it. Why would we support building Akka with something else than 2.11.8 (or latest 2.11.x) and 2.12.0-M5 (or latest 2.12.x)? Why maintain the old 2.12 milestones here? They will probably not compile anyway. Why not make scalatest 3.0.0 the default case, i.e. for Scala 2.11.8?